doc/user/sandbox/siri/SiriGooglePubSubUpdater.md

```diff
@@ -32,6 +32,7 @@ of the `router-config.json`.
 |------------------------------------------------------------|:----------:|----------------------------------------------------------------------------------|:----------:|---------------|:-----:|
 | type = "siri-et-google-pubsub-updater"                     |   `enum`   | The type of the updater.                                                         | *Required* |               |  1.5  |
 | [dataInitializationUrl](#u__12__dataInitializationUrl)     |  `string`  | URL used to download over HTTP the recent history of SIRI-ET messages.           | *Optional* |               |  2.1  |
+| detailedMetrics                                            |  `boolean` | If detailed metrics should be collected.                                         | *Optional* | `false`       |  2.1  |
```

Comment:
In which case would you not want these metrics? And isn't the provider added only when they are actually supplied in the source data?

Reply:
Agree this could be a bit more specific. I'll add some detail.

Reply:
I meant more: do we even need this setting?

Reply:
Oh I see. One scenario when you might not want them is if you have _very_ many providers. Combining tags like this easily leads to an explosion in the number of metrics, so I thought it could be useful to turn it off. This was also in the original suggestion in the issue.